ocaml/idl/datamodel_host.ml

```diff
@@ -1744,6 +1744,34 @@ let set_https_only =
       ]
     ~allowed_roles:_R_POOL_OP ()
 
+let install_rpmgpgkey =
+  call ~name:"install_rpmgpgkey"
+    ~doc:"Install an RPM GPG public key onto the host."
+    ~params:
+      [
+        (Ref _host, "self", "The host")
+      ; (String, "name", "The name of the RPM GPG public key")
+      ; (String, "pubkey", "The RPM GPG public key")
+      ; (String, "fingerprint", "The fingerprint of the RPM GPG public key")
+      ]
+    ~allowed_roles:(_R_POOL_OP ++ _R_CLIENT_CERT)
+    ~lifecycle:[] ()
```

Comment:
shouldn't this one be pool_internal, like uninstall?

Reply:
This will be used by client to determine if a key (may with same name) has been changed.

Reply:
I'd rather not override an install call to have 2 functionality, why can't a client check if a key has changed by querying the API for all the gpg_keys and checking their names and fingerprints?

Reply:
Ah. I see. I had a mistake in my mind.